packages/page-accounts/src/modals/CreateAccountInputs.tsx

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017-2021 @polkadot/app-accounts authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { useCallback } from 'react';
+
+import { Input, Modal } from '@polkadot/react-components';
+import { useTranslation } from '@polkadot/react-components/translate';
+
+import PasswordInput from './PasswordInput';
+
+interface AccountName {
+  name: string;
+  isNameValid: boolean;
+}
+
+interface AccountPassword {
+  password: string;
+  isPasswordValid: boolean;
+}
+
+interface Props {
+  name: AccountName;
+  setName: (value: AccountName) => void;
+  setPassword: (value: AccountPassword) => void;
+  _onCommit: () => void;
```

Comment:
```suggestion
  onCommit: () => void;
```

As above, not sure why this.

Reply:
✅